inc/namespace.php

```diff
@@ -187,6 +190,10 @@ function disable_no_cache_headers_on_admin_ajax_nopriv() {
  * Load the db dropin.
  */
 function load_db() {
+	$config = get_config();
```

Comment:
So as per the above comment, I think this should go in the `load_platform()` function.

Reply:
Also done in e8046bc2df8b733c678d63abdeb16c93bb2a7afa